frontend/tests/services/withFeatureFlag.test.tsx

```diff
@@ -53,19 +65,14 @@ describe("WithFeatureFlag", () => {
       "searchOff",
       onEnabled,
     );
-    render(<WrappedComponent searchParams={searchParams} />);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const component = await WrappedComponent({
+      searchParams: searchPromise(searchParams),
+    });
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    render(component);
     expect(onEnabled).toHaveBeenCalledTimes(1);
   });
-  it("does not call onEnabled during wrapped component render when feature flag is not enabled", () => {
```

Comment:
was this one too hard to rework?

Reply:
I removed it as I was troubleshooting, re-added it.